### PR TITLE
Add ssh key block storage

### PR DESCRIPTION
**Pull request by aajdinov**
Added block storage and ssh key support.

### CODE REVIEW

File: lib/chef/knife/oneandone_block_storage_list.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          block_storages << blks['datacenter']['country_code']
+          block_storages << (blks['server'].nil? ? 'nil' : blks['server']['id'])
+          block_storages << (blks['server'].nil? ? 'nil' : blks['server']['name'])
+        end
+
+        puts ui.list(block_storages, :uneven_columns_across, 11)
+      end
+    end
+  end
+end
```
Comment (nurfet-becirevic) [inline lib/chef/knife/oneandone_block_storage_list.rb]: The cmd output doesn't look very nice with so many data. You should remove some of them. Perhaps to remove `Description`, `Datacenter ID`, `Datacenter Location`, and rename `Datacenter Country Code` just to `Data Center`.
image

File: README.md [hunk 2]
```diff
@@ -130,6 +130,30 @@ Starts one or more servers.
...
+Deletes one or more block storages.
+
+### knife oneandone block storage attach
+
+Attaches a block storage to a server.
+
+### knife oneandone block storage detach
+
+Detaches a block storage from a server.
+
```
Comment (nurfet-becirevic) [inline README.md]: ssh key commands should be documented.
```
knife oneandone ssh key create (options)
knife oneandone ssh key delete SSH_KEY_ID [SSH_KEY_ID] (options)
knife oneandone ssh key list
knife oneandone ssh key rename (options)
```

**Comment by aajdinov** [comment]
@nurfet-becirevic I've made the requested changes. Do you have any other concerns, or are you green-lighting the PR to be merged?

**Comment by nurfet-becirevic** [comment]
```
block_storages << (blks['server'].nil? ? 'nil' : blks['server']['id'])
block_storages << (blks['server'].nil? ? 'nil' : blks['server']['name'])
```

```
ssh_keys << (ssh['description'].nil? ? 'nil' : ssh['description'])
```

I would replace `nil` with empty string as in

```
ip_list << (ip['assigned_to'].nil? ? '' : ip['assigned_to']['name'])
```

**Comment by aajdinov** [comment]
@nurfet-becirevic Thank you for the review. I've made the requested changes.

**Pull request merged**